src/components/molecules/event/eventButton/EventButton.tsx

```diff
@@ -168,20 +173,27 @@ const AuthEventButton: React.FC<AuthButtonProps> = ({
           {buttonText}
         </Button>
       )}
-      {lateCancellation && (
+      {cancellation && (
         <Modal
-          title="Er du sikker på at vil melde deg av?"
+          title="Er du sikker på at du vil melde deg av?"
           minWidth={25}
-          setIsOpen={setLateCancellation}>
+          setIsOpen={setCancellation}>
           <div className={styles.cancellationWrapper}>
             <div className={styles.cancellationMessageContainer}>
-              <p>
-                Arrangementet begynner om under 24 timer, og avmelding så nærme
-                arrangement start vil medføre en merknad. 2 eller flere
-                merknader vil gi nedsatt prioritet på andre arrangementer ut
-                semesteret. Har du gyldig grunn ta kontakt med{' '}
-                <a href="mailto:">{event?.host ?? 'post.td.uit.no'}</a>
-              </p>
+              {lateCancellation && (
```

Comment:
Think we can do `{lateCancellation ? (...) : (...)}` instead of `{lateCancellation && (...)}`
`{!lateCancellation && (...)}`

Reply:
Looks more elegant, will do!